inginious/frontend/plugins/manual_scoring/pages/students_list.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of UNCode. See the LICENSE and the COPYRIGHTS files for
+# more information about the licensing of this file.
+
+""" User list for a task in manual scoring page """
+
+from collections import OrderedDict
+from inginious.frontend.pages.course_admin.utils import INGIniousAdminPage
+from inginious.frontend.plugins.manual_scoring.pages.constants import get_render_path, get_use_minify
+
+base_renderer_path = get_render_path()
+
+
+def create_student_dict(user_list):
+    """ Organize student's data in a dictionary """
+    data = OrderedDict()
+    for user in user_list:
+        data[user["username"][0]] = {"username": user["username"][0],
+                                     "realname": user["realname"], "grade": user["grade"]}
+    return data
+
+
+class StudentsListPage(INGIniousAdminPage):
+    """ List users for a specific task """
+
+    def GET_AUTH(self, course_id, task_id):
+        """ Get request """
+        course, task = self.get_course_and_check_rights(course_id, task_id)
+        self.add_css_file()
+        return self.render_page(course, task_id, task)
+
+    def render_page(self, course, task_id, task):
+        """ Get all data and display the page """
+        task_name = course.get_task(task_id).get_name(self.user_manager.session_language())
```

Comment:
I think you already receive the task object, so it would be something like this:

```suggestion
        task_name = task.get_name(self.user_manager.session_language())
```